metacat-common-server/src/main/java/com/netflix/metacat/common/server/monitoring/Metrics.java

```diff
@@ -120,7 +126,8 @@
     enum Type {
         counter,
         gauge,
-        timer
+        timer,
+        distributionSummary
```

Comment:
To track the historical distribution of events ( here is for the partition operations )